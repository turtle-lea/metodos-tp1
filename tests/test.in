5.25 20 0.00001 1 
#
