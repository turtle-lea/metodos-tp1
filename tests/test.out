5.25
3 3.75
2.375 0.390625
2.29276 0.00676346
2.29129 2.38419e-06
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
2.29129 0
